.ci/prepare_env_utils.sh

```diff
@@ -40,6 +40,7 @@ function installDependencies() {
 function checkAllCreds() {
 	set -x
 	CREDS_NOT_SET="false"
+	OC_VERSION="3.10.85"
```

Comment:
why specific OC_VERSION is set ?

Reply:
I didn't find any place where it would be set from the config or machine. This is the latest version of the OC binary for `3.10.x` openshift which is running on dev cluster.

Reply:
ok, but why it needs to be defined explicitly ?

Reply:
it's used in the download URL for OC binary. I've discussed it with @Katka92 and found out I've set it at a wrong place :) Will fix it